kv/kvStore.go

```diff
@@ -0,0 +1,34 @@
+package kv
+
+import (
+	"errors"
+	"fmt"
+)
+
+const MaxMem = 1 << (10 * 3) // 1 GB
+const defPath = "."          // create in local directory
```

Comment:
Didn't realise what `def` stood for at first.
```suggestion
const defaultPath = "."          // create in local directory
```